### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by PeterDMadrid**
**OVERVIEW**

- Fix Routing from dashboard and profile page
- Added validation on login if user exist
- Added loader to disable post/share button, can no longer spam

https://github.com/xReddskie/microblog-ojt2/assets/91584922/2daa794b-f1cc-43d1-92e4-720292fd1ce2

### CODE REVIEW

File: src/routes/web.php [hunk 1]
```diff
@@ -24,8 +23,13 @@
 // Resend Email
 Route::get('/resend-email', [AuthController::class, 'resendEmailVerification']);
 
-// Dashboard Controller
-Route::get('/dashboard', [DashboardController::class, 'dashboard'])->middleware('auth')->name('dashboard');
+//Authh
```
Comment (hipe01mark) [inline src/routes/web.php]: Remove comment

File: src/routes/web.php [hunk 2]
```diff
@@ -24,8 +23,13 @@
...
+Route::middleware('auth')->group(function () {
+    // Dashboard
```
Comment (hipe01mark) [inline src/routes/web.php]: Remove comment

File: src/routes/web.php [hunk 3]
```diff
@@ -24,8 +23,13 @@
...
+    Route::get('/dashboard', [DashboardController::class, 'dashboard'])->name('dashboard');
+    // Profile Page
```
Comment (hipe01mark) [inline src/routes/web.php]: Remove comment

File: src/routes/web.php [hunk 4]
```diff
@@ -24,8 +23,13 @@
...
+    Route::get('/profile-page', [ProfileController::class, 'profilePage'])->name('profile-page'); 
+});
 
 //Posts Routes
```
Comment (hipe01mark) [inline src/routes/web.php:31]: Remove comment

File: src/routes/web.php (hunk 2 above)
Comment (PeterDMadrid) [reply src/routes/web.php]: Done

File: src/routes/web.php (hunk 3 above)
Comment (PeterDMadrid) [reply src/routes/web.php]: Done

File: src/routes/web.php (hunk 4 above)
Comment (PeterDMadrid) [reply src/routes/web.php:31]: Done

**Review by hipe01mark — APPROVED** [review]
GEFN. Kindly remove other comments next time.

**Pull request merged**